configs/default/AIRB-AIRBOTF4V2.config

```diff
@@ -0,0 +1,99 @@
+# Betaflight / STM32F405 (S405) 4.2.1 Jul 19 2020 / 06:18:01 (caa0d683c) MSP API: 1.43
+
+board_name AIRBOTF4V2
+manufacturer_id AIRB
+
+# resources
+resource BEEPER 1 C05
+resource MOTOR 1 C08
+resource MOTOR 2 B06
+resource MOTOR 3 C09
+resource MOTOR 4 B07
+resource LED_STRIP 1 A15
+resource SERIAL_TX 1 A09
+resource SERIAL_TX 2 A02
+resource SERIAL_TX 3 C10
+resource SERIAL_TX 4 A00
+resource SERIAL_TX 5 C12
+resource SERIAL_TX 6 C06
+resource SERIAL_RX 1 A10
+resource SERIAL_RX 3 C11
+resource SERIAL_RX 4 A01
+resource SERIAL_RX 6 C07
+resource I2C_SCL 1 B08
+resource I2C_SDA 1 B09
+resource LED 1 B12
+resource SPI_SCK 1 A05
+resource SPI_SCK 2 B13
+resource SPI_SCK 3 B03
+resource SPI_MISO 1 A06
+resource SPI_MISO 2 B14
+resource SPI_MISO 3 B04
+resource SPI_MOSI 1 A07
+resource SPI_MOSI 2 B15
+resource SPI_MOSI 3 B05
+resource ADC_BATT 1 C00
+resource ADC_CURR 1 C01
+resource FLASH_CS 1 A03
+resource OSD_CS 1 C15
+resource GYRO_CS 1 D02
+resource GYRO_CS 2 C04
+
+# timer
+timer A15 AF1
+# pin A15: TIM2 CH1 (AF1)
+timer A08 AF1
+# pin A08: TIM1 CH1 (AF1)
+timer C08 AF3
+# pin C08: TIM8 CH3 (AF3)
+timer B06 AF2
+# pin B06: TIM4 CH1 (AF2)
+timer C09 AF3
+# pin C09: TIM8 CH4 (AF3)
+timer B07 AF2
+# pin B07: TIM4 CH2 (AF2)
+
+# dma
+dma ADC 1 1
+# ADC 1: DMA2 Stream 4 Channel 0
+dma pin A15 0
+# pin A15: DMA1 Stream 5 Channel 3
+dma pin A08 0
+# pin A08: DMA2 Stream 6 Channel 0
+dma pin C08 0
+# pin C08: DMA2 Stream 2 Channel 0
+dma pin B06 0
+# pin B06: DMA1 Stream 0 Channel 2
+dma pin C09 0
+# pin C09: DMA2 Stream 7 Channel 7
+dma pin B07 0
+# pin B07: DMA1 Stream 3 Channel 2
+
+# feature
+feature -RX_PARALLEL_PWM
+feature RX_SERIAL
+feature OSD
+
+# serial
+serial 0 64 115200 57600 0 115200
+
+# master
+set mag_hardware = NONE
+set baro_hardware = NONE
```

Comment:
If this setting is used, this means that users will not be able to use any barometer on this board, and if they open support requests about this they will be told they will have to buy a different flight controller if they want to use a magnetometer.